internal/compiler/expression_tree.rs

```diff
@@ -1108,6 +1116,7 @@ impl Expression {
             Expression::SolveLayout(..) => false,
             Expression::MinMax { lhs, rhs, .. } => lhs.is_constant() && rhs.is_constant(),
             Expression::EmptyComponentFactory => true,
+            Expression::DebugHook { .. } => false, // We want to be able to override this at any point in time!
```

Comment:
```suggestion
            Expression::DebugHook { .. } => false,
```

False is correct.

(if you want to put a comment, put it on the previous line like for the other expressions.)

Reply:
Will be gone in the next push.